src/containers/BooksList.js

```diff
@@ -0,0 +1,38 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ books }) => (
+  <table>
+    <tr>
+      <th>Book ID</th>
+      <th>Title</th>
+      <th>Category</th>
+    </tr>
+    {books.map(book => (
+      <Book
+        key={book.title}
+        bookId={book.id}
+        bookTitle={book.title}
+        bookCategory={book.category}
+      />
+    ))}
+  </table>
+);
+
+BooksList.propTypes = {
+  books: PropTypes.instanceOf(Array),
+};
+
+BooksList.defaultProps = {
+  books: [],
+};
+
+function mapStateToProps(state) {
+  console.log(state);
```

Comment:
- [ ] Kindly remove this console.log line. It shouldn't be in the final code. It's a good practice to avoid committing debugging code.